RELEASE.md

```diff
@@ -84,6 +84,16 @@ releases page.
 Visit https://github.com/topolvm/topolvm/releases to check
 the result.  You may manually edit the page to describe the release.
 
+Release Helm Chart
+------------------
+
+TopoLVM Helm Chart will be released independently.
+This will prevent the TopoLVM version from going up just by modifying the Helm Chart.
+
+Please change the version of [Chart.yaml](./charts/topolvm/Chart.yaml) when releasing Helm chart.
+When the change is merged into the main branch,
+it will be automatically released by [helm/chart-releaser-action](https://github.com/helm/chart-releaser-action).
+
```

Comment:
Since the Helm chart has its own version, I thought it best to separate it from the topolvm release.
If you have an opinion on this, please comment 🙏

Reply:
I agree to separate Helm chart and topolvm release version.